packages/tooling/fast-tagged-templates/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [0.1.0] - 2020-10-01
+
+
+### Added
+- Initial source
```

Comment:
This will automatically version *again* when we go to publish to 0.2.0 and the changelog will automatically be created. Do we want to rev to 0.2.0?

Reply:
That would only happen if this extension is treated as a regular node package. This extension should be excluded from the normal publishing workflow, and handled independently.